tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -85,6 +85,7 @@
 use OrderState;
 use Pack;
 use Page;
+use PHPUnit\Framework\Assert;
```

Comment:
Wrong namespace here

Reply:
Why? And I'm always open to suggestions but if your comments are always about pointing what's wrong with no alternative it increases the number of back and forth message and makes the both of use lose time ^^